src/models/user/stream.rs

```diff
@@ -237,17 +257,21 @@ impl UserStream {
             )
             .await?
             .map(|set| set.into_iter().map(|(user_id, _score)| user_id).collect()),
-            UserStreamSource::Pioneers => Self::try_from_index_sorted_set(
-                &USER_PIONEERS_KEY_PARTS,
-                None,
-                None,
-                skip,
-                limit,
-                SortOrder::Descending,
-                None,
+            UserStreamSource::Influencers => Influencers::get_influencers(
+                user_id,
+                Some(source_reach.unwrap_or(StreamReach::Wot(3))),
+                skip.unwrap_or(0),
+                limit.unwrap_or(10).min(100),
+                &timeframe.unwrap_or(Timeframe::AllTime),
+                preview.unwrap_or(false),
             )
             .await?
-            .map(|set| set.into_iter().map(|(user_id, _score)| user_id).collect()),
+            .map(|result| {
+                result
+                    .iter()
+                    .map(|influencer| influencer.id.clone())
+                    .collect()
+            }),
```

Comment:
I think the previous implementation was correct. We do not need to `clone()` the user pubky plus do not the create new structs

Reply:
https://github.com/pubky/pubky-nexus/pull/234#discussion_r1884585292

Reply:
yes, that depends in the `Influencers` struct 👍